### ISSUE TITLE
Change mentions of jimmidyson to minishift in the code and documents

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#235]
We have moved from https://github.com/jimmidyson/minishift to https://github.com/minishift/minishift. However there are still mentions of jimmidyson in the code and it does not make sense now.
```
$ git grep jimmidyson

cmd/minikube/cmd/config/view.go:For the list of accessible variables for the template, see the struct values here: https://godoc.org/github.com/jimmidyson/minishift/cmd/minikube/cmd/config#ConfigViewTemplate`)
docs/minishift_config_view.md:For the list of accessible variables for the template, see the struct values here: https://godoc.org/github.com/jimmidyson/minishift/cmd/minikube/cmd/config#ConfigViewTemplate (default "- {{.ConfigKey}}: {{.ConfigValue}}\n")
glide.lock:  repo: https://github.com/jimmidyson/viper.git
glide.yaml:  repo: https://github.com/jimmidyson/viper.git
pkg/minikube/update/update.go:  githubOwner = "jimmidyson"
pkg/version/version.go:// This is a private field and should be set when compiling with --ldflags="-X github.com/jimmidyson/minishift/pkg/version.version=vX.Y.Z"
pkg/version/version.go:// This is a private field and should be set when compiling with --ldflags="-X github.com/jimmidyson/minishift/pkg/version.openshiftVersion=vX.Y.Z"
```

**Comment by hferentschik** [comment]
Not all of this can just be changed. There is more to it than that. Some of these mentions are legit.

**Comment by LalatenduMohanty** [comment]
@hferentschik 
Apart from 
```
glide.lock:  repo: https://github.com/jimmidyson/viper.git
glide.yaml:  repo: https://github.com/jimmidyson/viper.git
```
These lines can be fixed I think.

```
cmd/minikube/cmd/config/view.go:For the list of accessible variables for the template, see the struct values here: https://godoc.org/github.com/jimmidyson/minishift/cmd/minikube/cmd/config#ConfigViewTemplate`)
docs/minishift_config_view.md:For the list of accessible variables for the template, see the struct values here: https://godoc.org/github.com/jimmidyson/minishift/cmd/minikube/cmd/config#ConfigViewTemplate (default "- {{.ConfigKey}}: {{.ConfigValue}}\n")
pkg/minikube/update/update.go:  githubOwner = "jimmidyson"
pkg/version/version.go:// This is a private field and should be set when compiling with --ldflags="-X github.com/jimmidyson/minishift/pkg/version.version=vX.Y.Z"
pkg/version/version.go:// This is a private field and should be set when compiling with --ldflags="-X github.com/jimmidyson/minishift/pkg/version.openshiftVersion=vX.Y.Z"
```

**Comment by jimmidyson** [comment]
Even the viper issue might be fixed now, can't remember. Yep - see https://github.com/spf13/viper/pull/205. Needs testing to double check though.

**Comment by hferentschik** [comment]
> Needs testing to double check though.

Jump, the last time I checked, it was still needed. Let's double check

**Comment by hferentschik** [comment]
Merged via pull request #361
